packages/lab/src/checkbox/CheckboxIcon.css

```diff
@@ -0,0 +1,59 @@
+/* Styles applied to checkbox icon */
+/* Styles applied to root component */
+.uitk-density-medium {
+  --checkbox-icon-size: 14px;
+}
+
+.uitk-density-touch {
+  --checkbox-icon-size: 18px;
+}
+
+.uitk-density-low {
+  --checkbox-icon-size: 16px;
+}
+
+.uitk-density-high {
+  --checkbox-icon-size: 12px;
+}
+
+.uitk-density-high .uitkCheckboxIcon-indeterminate {
+  --checkbox-density-bar-y: 5;
+  --checkbox-density-bar-height: 3px;
+}
+
+.uitkCheckboxIcon {
+  --checkbox-solid-color: var(--uitk-selectable-fill-selected);
```

Comment:
--uitk-selectable-background-selected (selectable doesnt have fill anymore ... when my pr is merged that is)